File: Python/Guess the Number.py
import random

def main():
    print("What do you want your highest number to be?\n(The higher the number, the harder it is.)")
    highest = input()

    number = random.randint(1 , int(highest))

    print("\nGuess the number! ( 1 through" , int(highest) , ") .")
    answer = input()

    if answer == int(number):
        print("You guessed correctly!")
        main()
        
    else:
        print("Wrong! The answer was " , int(number) , "!\n")
        main()
main()

File: C++/Practice (other)/Guess the Number/guessthenumber.cpp
#include <iostream>

int main() {

    srand(time(0));
    int response;
    
    std::cout << "\nWhat do you want your highest number to be? (The higher the number, the harder it is.)\n";
    std::cin >> response;

    int highest = (response + 1);

    int number = rand() % highest;
    int answer;

    std::cout << "Guess the number! (1 through " << highest << ")\n";
    std::cin >> answer;

    if (answer == number) {
        
        std::cout << "You guessed correctly!\n";
        main();
    }

    else {
        std::cout << "\nWrong! The number was " << number << "!\n";
        main();
    }

    return 0;
}